lib/absinthe/schema/provider.ex

```diff
@@ -25,6 +25,4 @@ defmodule Absinthe.Schema.Provider do
   @callback __absinthe_directives__(schema_identifier) :: Absinthe.Type.Directive.t()
 
   @callback __absinthe_interface_implementors__(schema_identifier) :: term
-
-  @callback __absinthe_prototype_schema__(schema_identifier) :: schema_identifier
```

Comment:
Making this part of the behaviour was a mistake. The prototype schema is only used for the _construction_ of the schema, it isn't used at runtime, and it can't depend on the schema already existing.